storage_bucket/delete_file.py

```diff
@@ -34,33 +27,21 @@ class DeleteFile(object):
     :return: Result[None, Exception]
     """
 
-    _get_bucket = GetBucket()
+    get_bucket = GetBucket()
```

Comment:
Why would you change "visibility" to the public from protected in here?

Reply:
Hmm,, it was just because it wasn't private in any other classes.